source/GarminSDView.mc

```diff
@@ -226,6 +226,9 @@ class SdDelegate extends Ui.BehaviorDelegate {
     if (Storage.getValue(MENUITEM_LOWDATAMODE) == null) {
       Storage.setValue(MENUITEM_LOWDATAMODE, 0);
     }
+    if (Storage.getValue(MENUITEM_O2SENSOR) == null) {
+      Storage.setValue(MENUITEM_O2SENSOR, 0);
+    }
```

Comment:
Is this not setting the default value to zero?   I thought we would have made the default value 1 so the sensor is enabled by default?

Reply:
You are right.